packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/includes.json

```diff
@@ -4,7 +4,7 @@
     "message": "Your password has been reset"
   },
   "action": {
-    "id": "passwordResetAccountRecovery-action-2",
-    "message": "Create a new account recovery key"
```

Comment:
@bcolsson this string was modified after your review based on UXC review of the emails. The original string was modified to avoid the text breaking over two lines in the button, but I mentioned that text would likely still break in other locales. FWIW, the text breaks between words even for German's extra long `Kontowiederherstellungsschlüssel erzeugen`

Reply:
Is there any way this could be improved?

Reply:
I'm not sure. The copy was already reduced compared to the existing string, so I think this is an improvement, even for longer languages. I haven't seen any breakage _within_ words so this might be acceptable?  Unless we want to simplify even further and use "Create a (new) key", and that might not even help if locales need to use the full term for clarity.

Open to suggestions if you have them @bcolsson

Reply:
Oh, I assumed lines were breaking within words. If the button is sized to fit the German `Kontowiederherstellungsschlüssel` on one line without overflowing/breaking then I think that's okay.